Faraday_Proxy_Tools/FaradayIO/deviceconfig.py

```diff
@@ -163,13 +175,70 @@ def update_gps(self, gps_boot_bitmask, latitude_str, latitude_dir_str, longitude
 
         :return: Nothing
         """
+        # Setup check variables
+        lat_check = True
+        lat_dir_check = True
+        lon_check = True
+        lon_dir_check = True
+        alt_check = True
+        alt_units_check = True
+
+        # Check Latitude formatting
+
         lat_check = len(latitude_str) <= self.MAX_GPS_LATITUDE_LEN
         lat_dir_check = len(latitude_dir_str) <= self.MAX_GPS_LATITUDE_DIR_LEN
```

Comment:
OK this is a bit ridiculous, this ticket and pull was simple for a specific issue and for a cleaning of BSL not a complete implementation of config checking.

Making an enhancement issue and moving on...